src/sentry/static/sentry/app/components/tooltip2.jsx

```diff
@@ -0,0 +1,232 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styled from 'react-emotion';
+import PropTypes from 'prop-types';
+
+import {Manager, Reference, Popper} from 'react-popper';
+import {domId} from 'app/utils/domId';
+
+class Tooltip2 extends React.Component {
+  static propTypes = {
+    /**
+     * Disable the tooltip display entirely
+     */
+    disabled: PropTypes.bool,
+    /**
+     * The content to show in the tooltip popover
+     */
+    title: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
```

Comment:
This can just be `PropTypes.node`, which is pretty much synonymous here.

Reply:
Thanks I didn't know about PropTypes.node